app/user-setup/index.tsx

```diff
@@ -0,0 +1,215 @@
+                    Display name
+                </Text>
+                <Controller
+                    control={control}
+                    name="name"
+                    render={({ field: { onChange, value } }) => (
+                        <TextInput
+                            className="border-hairline h-14 rounded-md border-black px-3 py-1.5 text-lg text-gray-900"
+                            placeholder="Enter Display Name"
+                            onChangeText={onChange}
+                            value={value}
+                        />
+                    )}
+                />
+                {errors.name && (
+                    <Text className="mt-1 text-red-500">
+                        {errors.name.message}
+                    </Text>
+                )}
+            </View>
+            <View className="flex-col">
+                <View className="mb-2 mt-4 flex-row justify-start">
+                    <Text className="text-xl font-extrabold">Campus </Text>
+                    <Text className="text-xl color-gray-400">
+                        (preferred location)
+                    </Text>
+                </View>
+                {CampusEnum.options.map((campus, _) => {
+                    return (
+                        <TouchableOpacity
+                            key={campus}
+                            className={clsx(
+                                "my-1 grow justify-end rounded-lg px-2 py-3",
+                                campus === campusSelected
+                                    ? "border border-blue-500"
+                                    : "border-hairline border-dashed bg-white",
+                            )}
+                            onPress={() => handleSetCampus(campus)}
+                        >
+                            <Text
+                                className={clsx(
+                                    "text-center text-lg",
+                                    campus === campusSelected
+                                        ? "font-extrabold color-blue-400"
+                                        : "font-semibold color-black",
+                                )}
+                            >
+                                {campus}
+                            </Text>
+                        </TouchableOpacity>
+                    );
+                })}
+            </View>
+
+            <TouchableOpacity
+                className="mt-16 h-12 w-full justify-center rounded bg-blue-400 px-4 py-2"
+                onPress={handleSubmit(handleSetupUser)}
+                disabled={isLoading}
+            >
+                {isLoading ? (
+                    <ActivityIndicator className="w-full color-white" />
+                ) : (
+                    <Text
+                        className="text-center text-xl font-bold color-white"
+                        style={{ textAlignVertical: "center" }}
```

Comment:
instead of this, we can have a styling for the parent element to have

`items-center`

Reply:
it's the text jud mismo nga icenter sya vertically within the text component, kay items center is mag ani ra sya:
![image](https://github.com/user-attachments/assets/7318cefc-b22f-4b46-becc-63fe47adceb3)

Reply:
got it, quirky shit from react native